feature/announcement/src/main/java/io/github/droidkaigi/confsched2022/feature/announcement/Announcements.kt

```diff
@@ -220,6 +257,19 @@ fun FullScreenLoading(
     }
 }
 
+@Composable
+fun SnackBarWithError(
+    snackBarData: SnackbarData
+) {
+
+    Snackbar(
+        containerColor = MaterialTheme.colorScheme.primaryContainer,
+        actionColor = MaterialTheme.colorScheme.primary,
+        contentColor = MaterialTheme.colorScheme.onPrimaryContainer,
```

Comment:
Colors, etc. are tentative.
They will need to be changed when the design comes out.